statsmodels/tsa/vector_ar/tests/JMulTi_results/parse_jmulti_var_output.py

```diff
@@ -0,0 +1,421 @@
+import re
```

Comment:
There's a lot of work going on here to parse these results files.  Would it make sense to parse them once and commit them in a directly-importable form?

Related: the thoroughness is fantastic, but the large number of text files is daunting.  I imagine reviewers wouldn't be so reticent to jump in if it didn't say "208 files" up at the top of the window.